search-service/src/main/kotlin/com/egm/stellio/search/service/TemporalEntityAttributeService.kt

```diff
@@ -405,4 +402,33 @@ class TemporalEntityAttributeService(
     private var rowToId: ((Row) -> UUID) = { row ->
         row.get("id", UUID::class.java)!!
     }
+
+    suspend fun checkEntityAndAttributeExistence(
```

Comment:
I was more thinking of doing a SQL request and returning a generic message ("Either entity id or attribute name does not exist").

But yes, why not go for a more fine grained message (and a better error message) :) 

In this case, what about such a SQL query:

```sql
select exists(select 1 from temporal_entity_attribute where entity_id = :entity_id) as entityExists, 
       exists(select 1 from temporal_entity_attribute where entity_id = :entity_id and attribute_name = :attribute_name) as attributeNameExists;
```

Reply:
Just check that the entity exists? Because if it exists is that it's the attribute that does not exist?